tests/test_data/study_es_0_inc/data_methylation_hm27.txt

```diff
@@ -4,7 +4,6 @@ ACAP3	116983	0.022	0.681	0.790
 ATAD3A	55210	0.229	0.946	0.439
 ATAD3B	83858	0.885	0.707	0.664
 ATAD3C	219293	0.660	0.315	0.694
-#AURKAIP1	54998
```

Comment:
historically study_es_0 has always passed validation. Were these lines also there? What changed?

Reply:
study_es_0 has not passed validation before this fix. Remember automatic tests do not use this data, but you must load this study manually to see whether everything works. That's how I missed it.

Reply:
I think there is a misunderstanding: "_es_0" stands for "exit status 0" meaning "passes validation". So I'm wondering what change caused es_0 to start failing.

Reply:
I think the python validation script did not like the comment to be in the middle of the file.
Please also note, it's not the study_es_0 study folder, but the incremental upload data set.

Reply:
so `#AURKAIP1` is specific to the incremental folder (i.e. you made this addition) and is _not_ there in the original study_es_0 folder?

Reply:
update: Ruslan added this line. Now he is removing it.